schema/test/unit/computed_columns/form_change_holdback_amount_to_date_test.sql

```diff
@@ -1,13 +1,17 @@
 begin;
 
-select plan(1);
+select plan(3);
 
 /** SETUP **/
 
 insert into cif.project_revision(id, change_status)
   overriding system value
   values (1, 'pending');
 
+insert into cif.project_revision(id, change_status)
+  overriding system value
+  values (2, 'pending');
+
 insert into cif.form_change(
```

Comment:
suggest adding an archived value in here somewhere because filtering out archived is new in this test